composer.json

```diff
@@ -22,7 +22,8 @@
         "php-parallel-lint/php-parallel-lint": "1.3.0",
         "phpunit/dbunit": "1.3.2",
         "zf1s/phpunit": "3.7.39",
-        "staabm/annotate-pull-request-from-checkstyle": "1.5.0"
+        "staabm/annotate-pull-request-from-checkstyle": "1.5.0",
+        "rector/rector": "^0.12.12"
```

Comment:
since this project doesn't use compoer.lock, anchors not allowed. use specific version instead

Reply:
otoh. this requires specific version of php, so version range used by this project, this is not installable on all versions.

you probably need to remove composer.json changes

Reply:
Removed rector/rector